.agent/workflows/release-process.md

```
@@ -0,0 +1,470 @@
+# Security audit
+npm audit
+safety check        # Python
+go mod verify       # Go
+composer audit      # PHP
+```
+
+### 4. Build and Test
+
+Build the release artifacts:
+
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)